Formula/kim-api.rb

```diff
@@ -19,13 +20,20 @@ class KimApi < Formula
 
   depends_on "cmake" => :build
   depends_on "doxygen" => :build
-  depends_on "gcc" # for gfortran
+  depends_on "gcc"
+
+  fails_with :clang do
+    build 1200
+    cause "Mixed compiler usage (clang, clang++, gfortran) results in incorrect behavior."
+  end
 
   def install
     args = std_cmake_args
     # adjust compiler settings for package
-    args << "-DKIM_API_CMAKE_C_COMPILER=/usr/bin/clang"
-    args << "-DKIM_API_CMAKE_CXX_COMPILER=/usr/bin/clang++"
+    args << "-DKIM_API_CMAKE_C_COMPILER=/usr/local/bin/gcc-10"
```

Comment:
Please don't hardcode the path to compiler like this. This won't work on non-default prefixes, or on Apple Silicon.

If you need a way to refer to the compiler from the `gcc` formula, look at the `dynare` formula for an example.

Reply:
Excellent!  Thank you for the pointer on how to do this correctly.